components/sidebar/sidebar.jsx

```diff
@@ -35,7 +35,6 @@ export default class Sidebar extends Component {
           height={ availableHeight } />
 
         <div className="sidebar__inner">
-          <img src="https://img.shields.io/badge/webpack-v2.3.3-blue.svg" />
```

Comment:
@simon04 maybe just update this one to be the dynamic version you used in the installation guide? Unless we want to bump this manually as new docs are added...